morpheus/io/deserializers.py

```diff
@@ -107,6 +107,13 @@ def read_file_to_df(file_name: str,
             df.index.name = ""
             df.sort_index(inplace=True)
 
+        if (filter_nulls):
+            df = filter_null_data(df)
+
+        return df
+    elif (mode == FileTypes.PARQUET):
+        df = df_class.read_parquet(file_name, **kwargs)
+
```

Comment:
Restructure this function so that it doesnt repeat the filter nulls sections. So the if blocks should set the variable `df` and then that falls into a single section for `if(filter_nulls)`